tests/test_translation_hybrid.py

```diff
@@ -117,3 +118,9 @@ class Article(Base):
             locale = sa.Column(sa.String)
 
         Article.name
+
+    def test_aliased_entities(self, session, City):
+        # Ensure that queried entities are taken from the alias
+        CityAlias = aliased(City)
+        query_str = str(session.query(CityAlias.name))
```

Comment:
No need for session testing here. Why not just test the compilation of ``CityAlias.name`` expression.

```python
str(aliased(City).name.compile())
```